src/PKSim.Infrastructure/ORM/Repositories/OntogenyRepository.cs

```diff
@@ -115,6 +184,29 @@ private Ontogeny forMolecule(OriginData originData, string molecule)
          return allFor(originData.Species.Name).FindByName(molecule) ?? new NullOntogeny();
       }
 
+      public DataRepository OntogenyToRepository(Ontogeny ontogeny, string containerName)
+      {
+         var dataRepository = new DataRepository {Name = PKSimConstants.UI.OntogenyFor(ontogeny.DisplayName, containerName)};
+         var xUnit = _displayUnitRetriever.PreferredUnitFor(_dimensionRepository.AgeInWeeks);
+         var yUnit = _displayUnitRetriever.PreferredUnitFor(_dimensionRepository.Fraction);
+
+         var pma = new BaseGrid(PKSimConstants.UI.PostMenstrualAge, _dimensionRepository.AgeInWeeks) {DisplayUnit = xUnit };
+         var mean = new DataColumn(dataRepository.Name, _dimensionRepository.Fraction, pma) {DisplayUnit = yUnit };
+         var std = new DataColumn(PKSimConstants.UI.StandardDeviation, _dimensionRepository.Fraction, pma) {DisplayUnit = yUnit};
+         mean.DataInfo.AuxiliaryType = AuxiliaryType.GeometricMeanPop;
+         std.AddRelatedColumn(mean);
```

Comment:
Confused. Shouldn't it be `mean.AddRelatedColumn(std)` ??

Reply:
Confused too. I swapped it to try and it crashes
In this case the goal is to display area.  So the idea is that it's not like for oba data where mean is the main curve with some extra info
In this case, the area is the main curve and the mean is the extra stuff